CHANGELOG.md

```diff
@@ -22,13 +22,18 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Update jenkins file to use updated docker image ([#189](https://github.com/opensearch-project/opensearch-py-ml/pull/189))
 - Updated documentation @dhrubo-os ([#98](https://github.com/opensearch-project/opensearch-py-ml/pull/98))
 - Updating ML Commons API documentation @AlibiZhenis ([#156](https://github.com/opensearch-project/opensearch-py-ml/pull/156))
+- Update model upload history -  sentence-transformers/all-mpnet-base-v2 (v.1.0.3)(BOTH) by @thanawan-atc ([#140](https://github.com/thanawan-atc/opensearch-py-ml/pull/140))
```

Comment:
This is just to show what it will look like. But I will remove everything before creating a final PR and merging to the main repo.
<img width="845" alt="Screenshot 2023-08-07 at 11 09 27 PM" src="https://github.com/opensearch-project/opensearch-py-ml/assets/106889996/02011735-ee6a-4bc5-9f45-e4f0bd093d62">